deployment/CP05MOAS-A6263_Deploy.csv

```diff
@@ -48,31 +48,31 @@ WJ20190228,,WJ20190228,,CP05MOAS-A6263-03-CTDAVN000,7,1,2019-03-01T07:53:00,2019
-AR-34,,AR-34,,CP05MOAS-A6263-05-ADCPAN000,8,1,2019-04-10T12:40:00,2019-04-11T00:25:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-ADCPAN-22076,40.201167,-71.09885,,,,abbreviated mission due to time constraints (AL-1 to AL-2)
-AR-34,,AR-34,,CP05MOAS-A6263-06-PARADN000,8,1,2019-04-10T12:40:00,2019-04-11T00:25:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-PARADN-50267,40.201167,-71.09885,,,,abbreviated mission due to time constraints (AL-1 to AL-2)
+AR34,,AR34,,CP05MOAS-A6263-00-ENG000000,8,1,2019-04-10T12:40:00,2019-04-11T00:25:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGCON-AUVPNR-06263,40.201167,-71.09885,,,,abbreviated mission due to time constraints (AL-1 to AL-2)
+AR34,,AR34,,CP05MOAS-A6263-01-FLORTN000,8,1,2019-04-10T12:40:00,2019-04-11T00:25:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-FLORTN-01645,40.201167,-71.09885,,,,abbreviated mission due to time constraints (AL-1 to AL-2)
+AR34,,AR34,,CP05MOAS-A6263-02-DOSTAN000,8,1,2019-04-10T12:40:00,2019-04-11T00:25:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-DOSTAN-02901,40.201167,-71.09885,,,,abbreviated mission due to time constraints (AL-1 to AL-2)
+AR34,,AR34,,CP05MOAS-A6263-03-CTDAVN000,8,1,2019-04-10T12:40:00,2019-04-11T00:25:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-CTDAVN-00424,40.201167,-71.09885,,,,abbreviated mission due to time constraints (AL-1 to AL-2)
+AR34,,AR34,,CP05MOAS-A6263-04-NUTNRN000,8,1,2019-04-10T12:40:00,2019-04-11T00:25:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-NUTNRN-00366,40.201167,-71.09885,,,,abbreviated mission due to time constraints (AL-1 to AL-2)
+AR34,,AR34,,CP05MOAS-A6263-05-ADCPAN000,8,1,2019-04-10T12:40:00,2019-04-11T00:25:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-ADCPAN-22076,40.201167,-71.09885,,,,abbreviated mission due to time constraints (AL-1 to AL-2)
+AR34,,AR34,,CP05MOAS-A6263-06-PARADN000,8,1,2019-04-10T12:40:00,2019-04-11T00:25:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-PARADN-50267,40.201167,-71.09885,,,,abbreviated mission due to time constraints (AL-1 to AL-2)
 WJ20190617,,WJ20190617,,CP05MOAS-A6263-00-ENG000000,9,1,2019-06-18T19:18:00,2019-06-19T19:31:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGCON-AUVPNR-06263,40.203167,-71.09655,,,,along-shelf survey
 WJ20190617,,WJ20190617,,CP05MOAS-A6263-01-FLORTN000,9,1,2019-06-18T19:18:00,2019-06-19T19:31:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-FLORTN-01108,40.203167,-71.09655,,,,along-shelf survey
-WJ20190617,,WJ20190617,,CP05MOAS-A6263-02-DOSTAN000,9,1,2019-06-18T19:18:00,2019-06-19T19:31:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-DOSTAN-01519,40.203167,-71.09655,,,,along-shelf survey
+WJ20190617,,WJ20190617,,CP05MOAS-A6263-02-DOSTAN000,9,1,2019-06-18T19:18:00,2019-06-19T19:31:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-DOSTAN-01520,40.203167,-71.09655,,,,along-shelf survey

```

Comment:
Looks good.  Associated checkout document .zip file will need to be revised - remove the 1519 cal doc and insert the 1520 cal doc. (3403-40001-00017-A.zip)